uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -205,6 +208,6 @@ <h4 class="modal-title">Leave Request</h4>
     <button type="button" class="btn btn-primary pull-left" ng-click="$ctrl.cancel()">
       Back
     </button>
-    <button type="submit" class="btn btn-primary" ng-disabled="!$ctrl.canSubmit()" ng-click="$ctrl.submit()">Save</button>
+    <button type="submit" class="btn btn-primary" ng-disabled="!$ctrl.canSubmit() || $ctrl.isView()" ng-click="$ctrl.submit()">Save</button>
```

Comment:
Add `$ctrl.isView`check inside `canSubmit ` function instead of mentioning in the UI. So that we can unit test it.